src/openrct2/localisation/Localisation.h

```diff
@@ -156,7 +151,7 @@ class Formatter
         {
             convertedValue = reinterpret_cast<uintptr_t>(value);
         }
-        set_format_arg_body(CurrentBuf, 0, convertedValue, sizeof(TSpecified));
+        std::memcpy(CurrentBuf, &convertedValue, sizeof(TSpecified));
```

Comment:
Perhaps we can avoid the C-style `memcpy`, e.g. by using:
```suggestion
        *static_cast<TSpecified*>(CurrentBuf) = convertedValue;
```

Reply:
That just results in an error:
```
/home/michael/Programma's/Code/CLionProjects/OpenRCT2/src/openrct2/core/../localisation/Localisation.h:154:10: error: invalid static_cast from type ‘uint8_t*’ {aka ‘unsigned char*’} to type ‘short unsigned int*’
  154 |         *static_cast<TSpecified*>(CurrentBuf) = convertedValue;
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Reply:
I suppose a `reinterpret_cast` is in order, then.

Reply:
Does not work either. I get a similar error, but involving `uintptr_t` this time.

Reply:
It should keep using std::memcpy for now, reinterpret_cast should be avoided.